metaci/repository/utils.py

```diff
@@ -1,14 +1,16 @@
 from django.conf import settings
-from github3 import login
+from cumulusci.core.github import get_github_api
 
+def get_github_api(repo):
+    github = get_github_api(settings.GITHUB_USERNAME, settings.GITHUB_PASSWORD)
```

Comment:
I think you'd better rename the local `get_github_api` function, because I'm not sure that this won't call itself.

Reply:
I opted to remove repository.utils.get_github_api in favor of the existing Repository.github_api method, but this did require manually constructing the repo api in migrations which can't access that method.  Tested running migration locally.